CONTRIBUTING.md

```diff
@@ -48,8 +45,13 @@ for any of the following plugins, we encourage you to submit it
    fetch from the master repository, not your clone, when running `git fetch`
    et al.)
 
-## Running the examples
 
+## Setting up tools
+
+There are scripts for many common tasks (testing, formatting, etc.) that will likely be useful in preparing a PR.
+See [plugin_tools](./script/tool/README.MD) for more details.
```

Comment:
Can you scrub this for all instances of `.MD` and replace them with `.md`? I just got a 404 from one of these links due to the incorrect capitalization.

Reply:
Good catch! Thanks! Fix is here: https://github.com/flutter/plugins/pull/3815